layouts/partials/ecosystem/registry/entry.html

```diff
@@ -141,6 +142,26 @@ <h4 class="alert-heading">Deprecated</h4>
                     {{ end -}}
                     {{- .description | markdownify -}}
                 </div>
+
+                <!-- New Tags section -->
+                <div class="tags mb-2">
+                    {{ if .tags }}
+                        <div>
+                            {{ range .tags }}
+                                <span class="badge bg-primary me-1">{{ . }}</span>
```

Comment:
```suggestion
                                <span class="badge bg-light me-1">{{ . }}</span>
```

Let's use a light color, also make sure the text color fits the background color.

Reply:
Also, when I click on a tag I want to have that tag used as my search term

Reply:
@svrnm Alright sir

Reply:
@svrnm Done, I added a search link, so the link structure directs the user to /search, appending the tag as a query parameter. This should trigger the search functionality on your search page. And the urlQuery function is used to ensure that the tag is properly URL-encoded.